services/graphql/src/graphql/definitions/app-user.js

```diff
@@ -358,6 +359,8 @@ input SendAppUserLoginLinkMutationInput {
   redirectTo: String
   "If provided, will use the matched application context when sending the login email."
   appContextId: String
+  "If provided, will be used to append additional information to the login link token payload"
```

Comment:
@Shinsina - these two lines appear to be switched...the deprecated message should be on line 362 yes?

Reply:
@brandonbk No, `additionalEventData` is being newly added, the deprecation message is regarding the old `fields` input (line 365 with deprecation message on 364), this message here is regarding `additionalEventData` and how it functions within the `SendAppUserLoginLinkMutation`